tests/framework/input_for_test_cc/fullprint_Pointset.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" ?>
-<data name="fullprint_Pointset" type="Pointset">
-  <input>useless_input</input>
-  <output>normal1,normal2,uniform1,uniform2p</output>
-  <inputFilename>fullprint_Pointset.csv</inputFilename>
```

Comment:
It looks like we lost the inputFilename; is that something we want to restore to the meta? I honestly have no idea.

Reply:
We do not need this anymore. In the past, we do need this when we load the data from csv, since that is the file name of the data that we want to retrieve.